### DISCUSSION TITLE
Feature Request: integrate LaTeX in normal text.

### DISCUSSION THREAD

**Discussion by LucaGawa** [xournalpp/xournalpp#4460] [Ideas]
It would be great if there were a possibility to write LaTeX formulas or symbols directly into the text field (e.g. with $ $) and not only have independent objects, which are not aligned and not in the same size.

**Comment by netlimpopo** [comment]
I agree. Possible workarounds are working with Unicode Symbols which cover a few symbols but not all every LaTex Smbol (vectors. BUT At least arrows, greek letters, basics set properties, a few fractions, Real/Complex numbers). ,
On Linux its Ctrl +Shift +U then according Unicode number e.g. 2224 or 2102 or 211D . On Windows there is a UnicodeInput Utility App for Instance. 
Leaving out Space on the textflield to later align the LaTex Field is cumbersome.
Writing purely in Tex enlarges the file size significantly, but is very often the better case. Also LaTex minipages can be used.
